packages/compose/package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "@wordpress/compose",
 	"version": "1.0.1",
-	"description": "WordPress Higher Order components",
+	"description": "WordPress higher-order components",
```

Comment:
I'd add "(HOC)" in brackets here; lots of people throw that acronym around and giving newbies context here for free is 👍